src/runtime/pay-flow-test.js

```diff
@@ -281,14 +285,15 @@ describes.realWin('PayCompleteFlow', {}, env => {
     jserrorMock.verify();
   });
 
-  it('should have valid flow constructed', () => {
+  it('should have valid flow constructed ******', () => {
     const purchaseData = new PurchaseData();
     const userData = new UserData('ID_TOK', {
       'email': 'test@example.org',
     });
     const entitlements = new Entitlements('service1', 'RaW', [], null);
     const response = new SubscribeResponse(
-        'RaW', purchaseData, userData, entitlements);
+        'RaW', purchaseData, userData, entitlements,
+        ProductType.SUBSCRIPTION, entitlements);
```

Comment:
entitlements is here twice. You need to remove one of them

Reply:
Removed.